### ISSUE TITLE
Enhancement: Call Multisig from Anonymous Proxy

### ISSUE THREAD

**Issue by joepetrowski** [polkadot-js/apps#3998]
I have an anonymous proxy that is a member of a multisig. I tried starting a transaction process by clicking on the three-dot button for the multisig address and choosing "Add proxy" (I want to set a NonTransfer proxy for the multisig). It did not have the "use a proxy for this transaction" option even though the anon proxy is in my accounts as a proxy. I clicked on "Sign and Submit" and it was hanging with the following alert:
![image](https://user-images.githubusercontent.com/25483142/98546849-9b8c4780-2297-11eb-906f-223d0648a7a5.png)

Accounts:
![image](https://user-images.githubusercontent.com/25483142/98547023-daba9880-2297-11eb-8e19-322bcf7022f8.png)

Authorize Transaction screen:
![image](https://user-images.githubusercontent.com/25483142/98547218-210ff780-2298-11eb-8653-579a66f79783.png)

**Comment by jacogr** [comment]
Yes, it quite possibly won't work atm, anonymous proies are not really being exploited not actually specifically looked at in the UI atm.

Thanks for logging, yes, it has been in the back of my mind for "look at it someday" but never made it onto the growing list of items to do.

**Comment by shawntabrizi** [comment]
If I had a magic wand, the experience I might imagine here is the following:

* Some way to add a proxy account to show up in the accounts list
* Able to use it in any of the normal UIs just like a drop down
* If I use it, it wraps whatever calls the UI would make inside of the proxy wrapper
* This would trigger the user to sign with extension using the main account which controls the proxy
* Some nice indicator in the extension showing you are signing with account A to trigger Proxy account B

At that point, it should feel seamless to use a proxy account like a normal account where you have the seed saved.

**Comment by jacogr** [comment]
Shawn, see -

![image](https://user-images.githubusercontent.com/1424473/101078514-95436f80-35a6-11eb-8025-b69f830b92e7.png)

It gets treated like any normal account, although the seed is not kept online. 

The caveat is that when adding the stash already needs have a proxy for an account you do have, i.e. the dialog above checks the address you specified against the accounts you do have. Once added and sending from it, it always triggers the signer to use the proxy instead of the stash.

So -

- it shows up as a normal account
- it gets used like any normal account, in all dropdowns
- when used it gets wrapped inside a proxy wrapper

**Comment by ap211unitech** [comment]
This issue should be fixed with https://github.com/polkadot-js/apps/pull/11433. There were some issues if we use proxy for signing multisig txs, now hopefully all will be fixed.
